src/sub-blocks/BackgroundCard/__stories__/BackgroundCard.stories.tsx

```diff
@@ -95,13 +103,49 @@ const WithUrlTemplate: StoryFn<{items: BackgroundCardProps[]}> = (args) => (
     </div>
 );
 
+const ControlPositionTemplate: StoryFn<BackgroundCardProps> = (args) => (
+    <Grid>
+        <ConstructorRow>
+            <BlockBase>
+                <CardLayout title="With controlPosition = content" animated={false}>
+                    {data.cardLayout.items.map((item, index) => (
+                        <BackgroundCard
+                            key={index}
+                            {...(item as Partial<BackgroundCardProps>)}
+                            {...args}
+                            controlPosition="content"
+                        />
+                    ))}
+                </CardLayout>
+            </BlockBase>
+            <BlockBase>
+                <CardLayout
+                    title="With controlPosition = footer"
+                    description="Please note that the controlPosition prop manages the position of buttons and links only when paddingBottom = default."
```

Comment:
We don't store texts in `.tsx` story files, but keep them in `data.json`. In case we want to translate stories  it would be a problem to pick all of the texts form code